ludwig/datasets/mixins/kaggle.py

```diff
@@ -0,0 +1,38 @@
+import os
+from pathlib import Path
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+DEFAULT_CACHE_LOCATION = str(Path.home().joinpath('.ludwig_cache'))
```

Comment:
Doesn't look like `DEFAULT_CACHE_LOCATION` is needed here.

Reply:
Removed